### PR TITLE
Reducing image footprint

### PR DESCRIPTION
**Pull request by anadahz**
Include cleanup script to reduce image disk size
Add localepurge package to free some space
Set dpkg config file to ignore documentation and translation files
Enable compression in apt's indexe

### CODE REVIEW

File: lepidopter-fh/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Delete /var/log files
+find /var/log -type f -delete
+
+# Remove the same set of files and directories
+echo "Removing documentation..." >&2
+find /usr/share/doc -depth -type f ! -name copyright|xargs rm || true
+find /usr/share/doc -empty|xargs rmdir || true
+rm -rf /usr/share/man /usr/share/groff /usr/share/info /usr/share/lintian \
+/usr/share/linda /var/cache/man
```
Comment (bassosimone) [inline lepidopter-fh/cleanup.sh]: Could you indent this line right some spaces to make it easier to read?

**Comment by bassosimone** [comment]
Overall this looks good to me. If you could apply the cosmetics fix suggested above, I will proceed with merging this diff. Thanks!

File: lepidopter-fh/cleanup.sh (hunk 1 above)
Comment (anadahz) [inline lepidopter-fh/cleanup.sh]: Thanks, I have added a indentation fix.

**Comment by anadahz** [comment]
Thank you @bassosimone for reviewing this PR!

**Pull request merged**